source/visionEnhancementProviders/screenCurtain.py

```diff
@@ -80,8 +80,8 @@ def canStart(cls):
 	def __init__(self):
 		super(VisionEnhancementProvider, self).__init__()
 		Magnification.MagInitialize()
-		Magnification.MagShowSystemCursor(False)
 		Magnification.MagSetFullscreenColorEffect(TRANSFORM_BLACK)
+		Magnification.MagShowSystemCursor(False)
 
 	def terminate(self):
 		super(VisionEnhancementProvider, self).terminate()

```

Comment:
Could you please store this line in a try block, and the call that restores the mouse in a finally block? This ensures that it is always executed.

Reply:
> Could you please store this line in a try block, and the call that restores the mouse in a finally block? This ensures that it is always executed.

Belt and suspenders.
Done.